oauth2_provider/migrations/0004_idtoken.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2 on 2019-04-16 14:36
```

Comment:
is this migration auto-generated?

Reply:
Yes, it is.

Reply:
Isn't that they all are autogenarated, is it? Didn't hear about how to write Django migration by hand.

Reply:
You can. You can pass `-n` for `makemigrations` to give the migration a name too.